OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/DoublePlayerPlayerViewController.h

```diff
@@ -0,0 +1,9 @@
+#import <UIKit/UIKit.h>
+#import "SampleAppPlayerViewController.h"
+
+@interface DoublePlayerPlayerViewController : SampleAppPlayerViewController
+
+@property (nonatomic, retain) IBOutlet UIView *videoView;
```

Comment:
change retain to weak. IBOutlets do not need to retain or maintain a strong reference to their view.